internal/testing/testdata/testdata.go

```diff
@@ -431,7 +431,7 @@ var (
 		Digest:  []string{"sha256:6ad5b696af3ca05a048bd29bf0f623040462638cb0b29c8d702cbb2805687388"},
 		Version: "sha256:6ad5b696af3ca05a048bd29bf0f623040462638cb0b29c8d702cbb2805687388",
 		Purl:    "pkg:oci/static@sha256:6ad5b696af3ca05a048bd29bf0f623040462638cb0b29c8d702cbb2805687388?repository_url=gcr.io/distroless/static&tag=nonroot",
-		Tags:    []string{"CONTAINER"},
+		Tags:    []string{"container"},
```

Comment:
this may potentially break the demo, since in the demo, we use the CONTAINER tag, which is also used in SPDX [i think](https://github.com/guacsec/guac/blob/main/pkg/ingestor/parser/spdx/parse_spdx.go#L68)...

that change would need to happen as well on SPDX side to lower case container and `setup.md` to match.

Reply:
Yup I will update